datasette/static/app.css

```
@@ -519,7 +519,7 @@ input[type="search"]::-webkit-search-results-decoration {
     display: none;
 }
 
-form input[type=submit], form button[type=button] {
+form input[type=submit], form button[type=button], .btn-primary, .btn-secondary{
```

nit, leave a blank space before the brace
```suggestion
form input[type=submit], form button[type=button], .btn-primary, .btn-secondary {
```